### PR TITLE
Mg26x unscheduled

### PR DESCRIPTION
**Pull request by gitytakahas**
This is the madgraph cards for the single LQ3 -> tau + b search
(hosted under EXO group).

The folder contains 
> t-channel process: pp -> tau tau (via LQ exchange)
> s-channel process: pp->LQ + tau -> LQ(tau + b) + tau
> pair prod. process: pp->LQLQ -> tau tau b b

I uploaded just one mass point first, to see if this is OK.
Once I can get green light, I will upload remaining mass points.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_s-channel-M200/SingleLQ3ToTauB_5f_madgraph_LO_s-channel-M200_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#*                                                          *
+#************************************************************
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_s-channel-M200/SingleLQ3ToTauB_5f_madgraph_LO_s-channel-M200_proc_card.dat]: please cleanup the datacards following https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

Comment (gitytakahas) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_s-channel-M200/SingleLQ3ToTauB_5f_madgraph_LO_s-channel-M200_proc_card.dat]: Thanks for your quick check. I just updated to the new one. Please check again.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,16 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200_proc_card.dat]: the import call is duplicated

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+     0.0     = polbeam1 ! beam polarization for beam 1
+     0.0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+     nn23lo1    = pdlabel     ! PDF set                                     
+     230000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200_run_card.dat]: this is NNPDF23 which is very old compared to the current suggested datasets.
do you want to compare this MC to 2016 or 2017 data?
please pick up the correct sub-directory 2017 or pre2017, which apparently doesn't exist in your branch.
please update your branch to the master of https://github.com/cms-sw/genproductions/

**Comment by gitytakahas** [comment]
Thank you Luca - 
I just committed new changes (https://github.com/cms-sw/genproductions/pull/1507/commits/ba84422b021ee54f7b53774eb06b493135dafba1)

Answering your question, below:

1) This MC will be compared to 2016 data.
I don't have specific preference for the LHAID, but looking at pre2017 example (https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/ChargedHiggs_heavy_TB_M650/ChargedHiggs_heavy_TB_M650_run_card.dat#L58) I chose 292000

2) I would like to run this cards within mg26x branch, as the gridpack generation fails in master branch (Madgraph fails to compile the process, while this runs successfully in mg26). Do you still want to have this cards in master branch ?

**Comment by perrozzi** [comment]
Ciao Yuta, as I said you need to update you branch before to upload these datacards because the folder structure has changed, cfr
https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production
in particular you need to use the pre2017/13TeV folder
also, before to go to mg26x it would be good to understand why the compilation fails. I suspect that if you update your genproductions repository to the cms-sw:genproductions/master 
let me know if you have troubles in doing this

**Comment by gitytakahas** [comment]
OK, I will close this PR and request the new one to the master branch.

**Comment by gitytakahas** [comment]
@perrozzi, all

Concerning to the failed compilation in master branch, 
this is the screenshot of the error message: 



Do you have any idea about this? (I have no idea ...)
The strange thing is that, this worked with 26x branch.

**Pull request closed without merging**